packages/sandbox/e2e/visual_regressions_flow.yml

```diff
@@ -1,8 +1,22 @@
 appId: ${APPID}
 ---
-- launchApp
-- tapOn: 'Sandbox'
-- tapOn: 'Foundation'
+- launchApp:
+    stopApp: true # stop the app before launching it
+- runFlow:
+    when:
+      true: ${PLATFORM == 'ios'}
+    file: subflow/stopApp.yaml # workaround to handle ios deeplink correctly to run the test
+- openLink:
+    link: exp://localhost:19000
```

Comment:
@judastheo good trick to avoid the Sandbox issue? thanks @ludwig-pro 🎉
Work for me on iOS and Android (just needed to launch it a second time, then all good)